src/components/side-panel/panel-header.js

```diff
@@ -199,7 +200,6 @@ const defaultActionItems = [
   {
     id: 'save',
     iconComponent: Save,
```

Comment:
add `label: 'Share'` here

Reply:
@heshan0131 I change this. Sorry, I'm trying to understand your code. Thanks for the opportunity to learn a lot on this project.

Reply:
Thank you for keep working on this. All good now, will approve